### PR TITLE
update doc for pr #3488(quantization speedup tool)

### PR DESCRIPTION
**Pull request by linbinskn**

### CODE REVIEW

File: docs/en_US/Compression/QuantizationSpeedup.rst [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+
+Introduction
+------------
+
+Deep learning network has been computational intensive and memory intensive 
+which increases the difficulty of deploying deep neural network model. Quantization is a 
+fundamental technology which is widely used to reduce memory footprint and speed up inference 
+process. Many frameworks begin to support quantization, but few of them support mixed precision 
+quantiation. Frameworks like `HAQ: Hardware-Aware Automated Quantization with Mixed Precision <https://arxiv.org/pdf/1811.08886.pdf>`__\, only support simulated mixed precision quantization which will
```
Comment (J-shang) [inline docs/en_US/Compression/QuantizationSpeedup.rst:12]: quantiation -> quantization?

**Review by J-shang — APPROVED** [review]

File: docs/en_US/Compression/QuantizationSpeedup.rst (hunk 1 above)
Comment (linbinskn) [reply docs/en_US/Compression/QuantizationSpeedup.rst]: Thanks, have fixed.

File: docs/en_US/Compression/QuantizationSpeedup.rst [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    output, time = engine.inference(data)
+
+
+Note that NNI also supports post-training quantization directly, please refer to complete examples for detail.
+
+
+For complete examples please refer to :githublink:`the code <examples/model_compress/quantization/mixed_precision_speedup_mnist.py>`.
+
+
+For more parameters about the class 'TensorRTModelSpeedUp', you can refer to :githublink:`the code <nni/compression/pytorch/speedup/quantization_speedup/integrated_tensorrt.py>`.
```
Comment (QuanluZhang) [inline docs/en_US/Compression/QuantizationSpeedup.rst:83]: better to refer to API doc instead of source code

Comment (linbinskn) [reply docs/en_US/Compression/QuantizationSpeedup.rst]: Yeah, have done.

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**